ezsnmp/ez.py

```diff
@@ -42,7 +46,10 @@ def snmp_get(
 
 
 def snmp_set(
-    oid: Union[str, Tuple[str, str]], value: Any, type=None, **session_kargs
+    oid: Union[str, Tuple[str, str]],
+    value: Any,
+    type: Optional[str] = None,
+    **session_kargs: Any,
```

Comment:
If I recall right `session_kargs` is of type `dict`. 

```text
One can think of the kwargs as being a dictionary that maps each keyword to the value that we pass alongside it. That is why when we iterate over the kwargs there doesn’t seem to be any order in which they were printed out.
``` 
https://www.geeksforgeeks.org/args-kwargs-python/

Reply:
Done in 
dd0c9ee